components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -951,6 +953,9 @@ void BraveWalletService::RecordWalletUsage(bool unlocked) {
   p3a_utils::RecordFeatureNewUserReturning(
       prefs_, kBraveWalletP3AFirstUnlockTime, kBraveWalletP3ALastUnlockTime,
       kBraveWalletP3AUsedSecondDay, kBraveWalletNewUserReturningHistogramName);
+  p3a_utils::RecordFeatureLastUsageTimeMetric(
```

Comment:
Will it conflict with https://github.com/brave/brave-core/pull/15952/files#diff-64f2c39a961fa9988e9b519e1b1ab653b97ec7089693e1f14e77503200165cd2L918 ?

Reply:
yeah it will. planning to rebase that pr once this one is merged